gulpfile.js/extras/craft/gulprev/twigextensions/GulpRevTwigExtension.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Craft;
+
+use Twig_Extension;
+use Twig_Filter_Method;
+
+class GulpRevTwigExtension extends Twig_Extension
+{
+    public function getName()
+    {
+        return 'gulpRev';
+    }
+
+    public function getFilters()
```

Comment:
If we rename this to `getFunctions` you could use this as a Twig function over a Filter. So:
`<link rel="stylesheet" href="{{  gulpRev('assets/stylesheets/app.css') }}">`

over

`<link rel="stylesheet" href="{{  'assets/stylesheets/app.css' | gulp_rev }}">`

Or we could include both functionality and call a protected method that returns the rev'd string.